eslint.config.mjs

```diff
@@ -0,0 +1,18 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import tseslint from "typescript-eslint";
+
+export default [
+  {
+    files: ["**/*.js"],
+    languageOptions: {
+      sourceType: "script",
+      globals: globals.browser
```

Comment:
I've run the linter and it was surfacing over 2000 problems... However, most of them are not actual problems. I know the scope of this story isn't to resolve the issues, but I think that the nature of the problems being identified (lots of "require is not defined" type errors) point to the configuration being incorrect.

The following suggestion resolves the vast majority of these. Combined with my other comment in the "ignores" section, it reduces the count to ~700.

```suggestion
      globals: {
        ...globals.browser,
        ...globals.node,
        ...globals.jest
      }
```